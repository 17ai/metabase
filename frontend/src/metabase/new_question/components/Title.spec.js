import React from "react";
import renderer from "react-test-renderer";

import Title from "./Title";

describe("Title", () => {
    it("should render correctly", () => {
        const tree = renderer
            .create(<Title>Hey it's me the title</Title>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
