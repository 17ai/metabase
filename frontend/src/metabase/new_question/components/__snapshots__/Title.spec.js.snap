exports[`Title should render correctly 1`] = `
<h1
  className="c-2E353B fs-25p63px fw-900">
  Hey it\'s me the title
</h1>
`;
